apps/chat/channels.py

```diff
@@ -680,7 +693,10 @@ def _transcribe_audio(self, audio: BytesIO) -> str:
         return "Unable to transcribe audio"
 
     def _get_bot_response(self, message: str) -> ChatMessage:
-        return self.bot.process_input(message, attachments=self.message.attachments)
+        chat_message = self.bot.process_input(message, attachments=self.message.attachments)
+        self.output_voice_provider_id = getattr(self.bot, "voice_provider_id", None)
```

Comment:
I still think it will be valuable to let the bot synthesize the voice, since it has the knowledge of which voice to use. In the `TopicBot` case, we use the `processor_experiment`. In the `Pipeline` bot, we use `voice_provider_id` and `output_synthetic_voice_id`. The only reason that `processor_experiment` is being used is to know which voice to use

Reply:
@SmittieC I'm not quite following what you're saying

Reply:
That seems reasonable